modules/ddl-and-loading/pages/managing-loading-job.adoc

```diff
@@ -257,4 +257,19 @@ Of the 6 valid lines,
 * Two lines (Lines 4 and 5) do not pass the `WHERE` clause.
 
 The errors that appear are the first line with an error in each batch in the loading job.
-Up to 100 sample lines showing errors appear in the log.
\ No newline at end of file
+Up to 100 sample lines showing errors appear in the log.
+
+== Handling Failures in Streaming and Batch Loading Jobs
+=== Batch (File) Loading Jobs
+* Auto-resume Mechanism: Starting from version 3.10, the auto-resume mechanism ensures that the job resumes from where it left off.
```

Comment:
Stylistically, it is better to have a blank line between Asciidoc/Markdown elements.